src/asm3/paymentprocessor/cardcom.py

```diff
@@ -24,7 +24,7 @@ def getInvoiceItems(self, records):
             yield {"InvoiceLines%d.Quantity" % index: 1}
             yield {"InvoiceLines%d.Price" % index: price} 
 
-    def tokenCharge(self, payref, item_description=""):
+    def tokenCharge(self, payref, item_description = "", installments = 1):
```

Comment:
Probably not a big deal as you're always going to pass installments, but your default type here is int, but you're passing str and it looks like it should be str for the JSON payload